provision.azure.yaml

```diff
@@ -1,3 +1,5 @@
 resources:
   topics:
-    - name: alert
+    - name: ffc-pay-alerts
```

Comment:
in line with what other Function app configs -- did not create any extra Topic, i.e. ffc-pay-ffc-pay-alerts as I perhaps thought (maybe thought it used repo name as prefix to this)

Reply:
For functions the name that goes here isn't as restrictive as the Kubernetes applications.

In those this value MUST match the expected environment variable and helm chart value.

eg if the value was `ALERT_TOPIC_ADDRESS` and `alertTopicAddress` then `alert` must be in this file for the integration tests in CI and dynamic PR deployment to be deployed with the right values.

There will be two pipelines generated from this line, one for CI, one for that dynamic deployment.

If the value remains as alert they would be.

`ffc-pay-alerts-pr3-alert` (lives life of PR)
`ffc-pay-alerts-pr3-b1-alert` (deleted once build completes)

As you've changed it you'll now get

`ffc-pay-alerts-pr3-ffc-pay-alerts` (lives life of PR)
`ffc-pay-alerts-pr3-b1-ffc-pay-alerts` (deleted once build completes)

There is a max name length with service bus for subscriptions so you don't want it to be too long.

I would change it back just because the change isn't really adding anything.

Reply:
aaahhhh gotcha, gotcha -- thanks for this, I really must remember what this value is for and how it relates to deployments namings.

Will revert back.